Example/STKWebKitViewController.xcodeproj/project.pbxproj

```diff
@@ -502,6 +502,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "STKWebKitViewController/STKWebKitViewController-Prefix.pch";
 				INFOPLIST_FILE = "STKWebKitViewController/STKWebKitViewController-Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```

Comment:
I didn´t choose a deployment target in the podspec, because then you can include this pod in your Podfile, even if your app is running on a lower deployment target than 8.0. Basically, Cocoapods sets the deployment target for this pod to 4.3, if nothing was declared. But yeah, you have to check the iOS version in your app before using STKWebKitViewController, then.
For me personally it would mean, that I cannot include the pod in the Podfile in many of my apps...so if there would be a way to get rid of this, it would be much appreciated.

Anyways, thank you very much for your effort up to this point!